examples/chain_client_examples/11_ExchangeMsgBatchCreateSpotLimitOrders.py

```diff
@@ -15,33 +15,25 @@ async def main() -> None:
     client = Client(network.grpc_endpoint, insecure=True)
 
     # load account
-    priv_key = PrivateKey.from_hex("5d386fbdbf11f1141010f81a46b40f94887367562bd33b452bbaa6ce1cd1381e")
+    priv_key = PrivateKey.from_hex("f9db9bf330e23cb7839039e944adef6e9df447b90b503d5b4464c90bea9022f3")
     pub_key =  priv_key.to_public_key()
     address = pub_key.to_address()
     subaccount_id = address.get_subaccount_id(index=0)
 
     # prepare trade info
     market_id = "0xa508cb32923323679f29a032c70342c147c17d0145625922b0ef22e955c844c0"
     fee_recipient = "inj1hkhdaj2a2clmq5jq6mspsggqs32vynpk228q3r"
-
-    orders = [
-        ProtoMsgComposer.SpotOrder(
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            fee_recipient=fee_recipient,
-            price=7.523,
-            quantity=0.01,
-            isBuy=True
-        ),
-        ProtoMsgComposer.SpotOrder(
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            fee_recipient=fee_recipient,
-            price=27.92,
-            quantity=0.01,
-            isBuy=False
-        ),
-    ]
+    orders = []
+    for order in range(5):
```

Comment:
Same with this one. My batch orders can be intepreted as "buy when INJ if price is 7.523 and sell INJ if price is 27.92". While the new example might confuse them why they have to add 5 similar orders to pay extra gas when we can do 1 order with quantity = 0.05.

Reply:
Yeah that was just an example, they could loop to make 5 orders for price x=x+1. I don't think they'd manually add this data for every single order given that they place 20 orders+ per market